src/components/user/uc/users.uc.js

```diff
@@ -6,6 +6,11 @@ const { facadeLocator } = require('../../../utils/facadeLocator');
 const errorUtils = require('../../../errors/utils');
 const { trashBinResult } = require('../../helper/uc.helper');
 
+const getSchoolIdOfUser = async (userId) => {
+	const user = await userRepo.getUser(userId);
```

Comment:
userRepo.getUser or getUserSchoolId with specific select in repo to pull only needed data?